x-pack/plugins/index_management/__jest__/client_integration/template_create.test.tsx

```diff
@@ -308,6 +308,47 @@ describe.skip('<TemplateCreate />', () => {
     });
   });
 
+  describe('saves the index template', () => {
```

Comment:
Note that this suite is skipped by https://github.com/elastic/kibana/issues/67833 and you'll have to unskip it to run it locally.